src/test/pull_hf_model_test.cpp

```diff
@@ -307,6 +339,39 @@ TEST(OvmsGraphConfigTest, positiveSomeChanged) {
     ASSERT_EQ(config.getServerSettings().hfSettings.graphSettings.draftModelDirName.has_value(), false);
 }
 
+TEST(OvmsGraphConfigTest, positiveTaskTextGen) {
```

Comment:
Move those tests to ovmsconfig_test -> they don't need to recompile when libgit2 changes